trieste/acquisition/function.py

```diff
@@ -163,9 +163,20 @@ def prepare_acquisition_function(
         self, dataset: Dataset, model: ModelInterface
     ) -> AcquisitionFunction:
         """
-        Need to sample possible min-values from our posterior. 
+        Need to sample possible min-values y* from our posterior. 
         To do this we implement a Gumbel sampler.
-        We approximate Pr(y*^hat<y) by Gumbel(a,b) then sample from Gumbel.
+        We approximate Pr(y*<y) by Gumbel(a,b) then sample from Gumbel.
+
+        The Gumbel distribution for minimas has a cumulative density function
```

Comment:
i'm curious about docs for research code. In other contexts, I wouldn't normally include the explanation of the implementation, as typically the only thing the user needs to know is what values they get back for any given input, but here, part of the importance of the function is how it works, so that researchers can compare algorithms

Reply:
I feel like it is useful? Let me know if you want me to get rid of it.

Reply:
no no need to, it was more of a musing than a suggestion